contents/templates/growth-dashboard/index.mdx

```diff
@@ -5,7 +5,7 @@ thumbnail: ../thumbnails/support-dashboard.png
 featuredImage: ../thumbnails/featured/growth-metrics.png
 showTitle: true
 filters: {
-    type: ["product","marketing"],
+    type: ["product","marketing","dashboard"],
```

Comment:
I think we should remove the 'Product' and 'Marketing' categories entirely, and focus templates in by feature - Dashboard templates, Survey templates. 

It'd just be more consistent, I think. Having a mix of feature and use-based terms in here could be confusing.

Reply:
removed